app/models/manager_refresh/graph.rb

```diff
@@ -10,6 +10,38 @@ def initialize(nodes)
       construct_graph!(@nodes)
     end
 
+    def to_graphviz(layers: nil)
+      node_names = {}
```

Comment:
I was expecting a `require "graphviz"` or something here, but don't see it.

Reply:
Not running the render at logging time, just printing text that's valid graphviz syntax.
I suppose there are gems that generate graphviz syntax, but can't make it much simpler, plus here I don't just want *any* valid syntax, I want particular that's also readable to a human just reading the log.

Reply:
Ahhh...makes sense.